### PR TITLE
Fixed #5765 form view do not extent text area

### LINKED ISSUE #5765: 🐛 Bug: form view do not extent text area
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

1. create text field
2. create form view
3. create share link
4. enter long text to the form using shared view

## Expected behavior

text ares should expand.

### Project Details

deployed on digitalocean server using docker

### Attachments

Screenshots

image

### PR DESCRIPTION
**Pull request by shashankvish**
## Change Summary

This change ensures that the nc-form-after-submit-msg text area no longer allows the text to overflow outside the container. Users can now enter and view large messages within the text area, with the ability to scroll through the content when necessary fixes #5765

## Change type

- [x] fix: form view do not extend text area

## Test/ Verification

To test and verify the fix, follow these steps:
- Compile and build the project to ensure you have the latest changes.
- Navigate to the area where the nc-form-after-submit-msg text area is present.
- Enter a large message that would normally cause the text to overflow.
- Verify that the text area now displays a scrollbar and the text does not extend beyond its boundaries.
- Scroll within the text area to confirm that you can view the entire content of the message.
- Repeat the test with different message lengths to ensure the fix handles various scenarios.

## Additional information / screenshots:

Screenshot (246)

### CODE REVIEW

File: packages/nc-gui/components/smartsheet/Form.vue [hunk 1]
```diff
@@ -777,13 +777,13 @@ watch(view, (nextView) => {
...
-            class="nc-form-after-submit-msg"
-            data-testid="nc-form-after-submit-msg"
-            @change="updateView"
+         <a-textarea
+        v-model:value="formViewData.success_msg"
+        :rows="3"
+         hide-details
+          class="nc-form-after-submit-msg"
+          data-testid="nc-form-after-submit-msg"
+          @change="updateView"
```
Comment (wingkwong) [inline packages/nc-gui/components/smartsheet/Form.vue:786]: Unexpected indentation. Please run lint or just replace the old one back.

File: packages/nc-gui/components/smartsheet/Form.vue [hunk 2]
```diff
@@ -838,20 +838,24 @@ watch(view, (nextView) => {
 </template>
 
 <style scoped lang="scss">
+
+
```
Comment (wingkwong) [inline packages/nc-gui/components/smartsheet/Form.vue:842]: I don't see the point of adding two new lines here. Please remove them.

File: packages/nc-gui/components/smartsheet/Form.vue [hunk 3]
```diff
@@ -838,20 +838,24 @@ watch(view, (nextView) => {
...
   :deep(.nc-field-remove-icon) {
     @apply opacity-100;
   }
 }
 
 .nc-input {
-  @apply appearance-none w-full !bg-white rounded px-2 py-2 my-2 border-solid border-1 border-primary border-opacity-50;
-
+  @apply appearance-none w-full !bg-white rounded px-2 py-2 my-2 border-solid border-1 border-primary border-opacity-50 overflow-auto;
+
```
Comment (wingkwong) [inline packages/nc-gui/components/smartsheet/Form.vue:851]: please remove this line.

File: packages/nc-gui/components/smartsheet/Form.vue [hunk 4]
```diff
@@ -838,20 +838,24 @@ watch(view, (nextView) => {
...
-
+  @apply appearance-none w-full !bg-white rounded px-2 py-2 my-2 border-solid border-1 border-primary border-opacity-50 overflow-auto;
+  
   :deep(input) {
     @apply !px-1;
   }
 }
 
+
+
```
Comment (wingkwong) [inline packages/nc-gui/components/smartsheet/Form.vue:858]: same as above. No need to add new lines.

File: packages/nc-gui/components/smartsheet/Form.vue [hunk 5]
```diff
@@ -863,6 +867,7 @@ watch(view, (nextView) => {
   }
 }
 
+
```
Comment (wingkwong) [inline packages/nc-gui/components/smartsheet/Form.vue:870]: same as above. No need to add a new line here.

**Review by wingkwong — CHANGES_REQUESTED** [review]
1. Please sign CLA
2. Please don't commit package-lock.json as the changes aren't related to this PR.

The expecting change should be just 1 line modified in 1 file.

File: packages/nc-gui/components/smartsheet/Form.vue (hunk 3 above)
Comment (shashankvish) [reply packages/nc-gui/components/smartsheet/Form.vue:851]: okay sir.

File: packages/nc-gui/components/smartsheet/Form.vue (hunk 1 above)
Comment (shashankvish) [reply packages/nc-gui/components/smartsheet/Form.vue:786]: Okay will do that.

**Comment by shashankvish** [comment]
Updated

**Comment by shashankvish** [comment]
> 1. Please sign CLA
> 2. Please don't commit package-lock.json as the changes aren't related to this PR.
> 
> The expecting change should be just 1 line modified in 1 file.

Already Signed the CLA but its not showing here.

**Comment by wingkwong** [comment]
> Already Signed the CLA but its not showing here.

Verified CLA. 

However, other comments are still not resolved. Namely 

- Please don't commit package-lock.json as the changes aren't related to this PR.
- Remove L854 - L855

The expecting change should be just 1 line modified in 1 file. Please request review again once they are resolved.

**Comment by shashankvish** [comment]
Updated by removing all the changes, please let me know if this is okay to merge to need some changes.

**Pull request closed without merging**